server/app/boot.rb

```diff
@@ -30,3 +30,5 @@
 Dir[__dir__ + '/jobs/**/*.rb'].each {|file| require file }
 
 Dir[__dir__ + '/services/**/*.rb'].each {|file| require file }
+
+Mongodb::Migrator.new.migrate
```

Comment:
#999 doesn't run migrations except when running puma, it's not running them before tests either.